dash_auth/auth.py

```diff
@@ -1,29 +1,45 @@
 from __future__ import absolute_import
 from abc import ABC, abstractmethod
 
+from dash import Dash
+from flask import Flask, request
+from werkzeug.datastructures import ImmutableDict
+from werkzeug.routing import Map, MapAdapter, Rule
+
+# Add PUBLIC_ROUTES in the default Flask config
+default_config = Flask.default_config
+Flask.default_config = ImmutableDict(
+    **default_config, **{"PUBLIC_ROUTES": Map([]).bind("")}
+)
+
 
 class Auth(ABC):
-    def __init__(self, app, authorization_hook=None, _overwrite_index=True):
+    def __init__(self, app: Dash, **_kwargs):
```

Comment:
We probably don't want to silently discard `**_kwargs`, can we just take this out and have it throw a `TypeError: Auth got an unexpected keyword argument <whatever>`? Seems like `authorization_hook` hasn't been used since we removed `PlotlyAuth`, and I don't see a use of `_overwrite_index` even there.